packages/@webex/plugin-meetings/test/unit/spec/members/index.js

```diff
@@ -342,7 +342,7 @@ describe('plugin-meetings', () => {
       });
     });
 
-    describe.only('#assignRoles', () => {
+    describe('#assignRoles', () => {
```

Comment:
this should be already fixed on latest beta

Reply:
Yeah, that change is gone now that I've updated to the latest beta